framework/tests/reflection/MethodTest.java

```diff
@@ -57,10 +57,6 @@ public void pass3() {
         try {
             Class<?> c = Class.forName("MethodTest$SuperClass");
             Method m = c.getMethod(str, new Class[] {});
-            // TODO: Should not fail -> enhance Value checker
-            // and remove the expected error
-
-            // :: error: (assignment.type.incompatible)
             @Sibling1 Object a = m.invoke(superClass, (@ReflectBottom Object[]) null);
```

Comment:
What's the relation between this assignment and the string concatenation?

Reply:
Line 56 isn't handled correctly, b/c the Value Checker didn't handle StringConcatenateAssignmentNode.

Reply:
Yeah, solved.